source/configuration_manual/mail_location/obox/dictmap.rst

```diff
@@ -186,6 +186,18 @@ Dictmap Parameters
 |                                 |                                                                              |
 |                                 | .. versionadded:: v2.3.10                                                    |
 +---------------------------------+------------------------------------------------------------------------------+
+| cleanup-uncertain               | If this setting is enabled and writes to Cassandra fail with uncertainty     |
+|                                 | (:ref:`dictmap_cassandra_uncertain_writes`) Dovecot attempts to clean up.    |
```

Comment:
Maybe this would be still a bit clearer?

When enabled: If a write to Cassandra fails with uncertainty (:ref:`dictmap_cassandra_uncertain_writes`), Dovecot attempts to clean it up: First...

Reply:
yes, changed it.